jkube-kit/config/resource/src/main/java/org/eclipse/jkube/kit/config/resource/ResourceServiceConfig.java

```diff
@@ -35,7 +36,7 @@
 public class ResourceServiceConfig {
 
   private JavaProject project;
-  private File resourceDir;
+  private List<File> resourceDirs;
```

Comment:
You could annotate this field with the `@Singular` annotation to make the builder more user-friendly.